From 813d7063e3d265ba7e625766a040b8ba9bb130a9 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Thu, 16 Aug 2012 11:18:00 +0200
Subject: [PATCH] patch: xdg-open

Squashed commit of the following:

commit 55380b90cece459e20d14e6da552abcf5ca54621
Author: Nils Philippsen <nils@redhat.com>
Date:   Fri Nov 19 12:14:17 2010 +0100

    use "xdg-open" instead of "netscape" to launch help browser
---
 src/xsane.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xsane.h b/src/xsane.h
index cf6111f..3d8caaa 100644
--- a/src/xsane.h
+++ b/src/xsane.h
@@ -251,7 +251,7 @@
 # elif defined(HAVE_OS2_H)
 #  define DEFAULT_BROWSER "netscape"
 # else
-#  define DEFAULT_BROWSER "netscape"
+#  define DEFAULT_BROWSER "xdg-open"
 # endif
 #endif
 
-- 
1.7.11.4

